### PR TITLE
Add library SCIP++

### PR DESCRIPTION
**Pull request by hedtke**
Specify library name and version:  **SCIP++/1.0.2**

I am the author of this library. It is based on SCIP which was added to conan-center recently

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by hedtke** [comment]
Recipe is almost copy&paste, despite the original repo having a recipe already, see #18876

**Comment by czoido** [comment]
Thanks a lot for the contribution @hedtke!

**Review by czoido — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

File: recipes/scippp/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.scm import Version
+from os.path import join
+
+
+class ScipPlusPlus(ConanFile):
+    name = "scippp"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeDeps"
+    exports_sources = "CMakeLists.txt", "source/*", "include/*"
```
Comment (SpaceIm) [inline recipes/scippp/all/conanfile.py:15]: `exports_sources` can be removed

Comment (hedtke) [reply recipes/scippp/all/conanfile.py:15]: see https://github.com/conan-io/conan-center-index/pull/19458

**Pull request merged**